pkgs/development/libraries/zchunk/default.nix

```diff
@@ -11,15 +11,15 @@
 
 stdenv.mkDerivation rec {
   pname = "zchunk";
-  version = "1.1.9";
+  version = "1.1.11";
 
   outputs = [ "out" "lib" "dev" ];
 
   src = fetchFromGitHub {
     owner = "zchunk";
     repo = pname;
     rev = version;
-    hash = "sha256-MqnHtqOjLl6R5GZ4f2UX1iLoO9FUT2IfZlSN58wW8JA=";
+    sha256 = "1x7nrrhcafml6sgqr5h73v3hlcyq2275npngik4j6kr7wlj9dsmg";
```

Comment:
```suggestion
    hash = "sha256-r+qWJOUnTyPJjM9eW44Q2DMKxx4HloyfNrQ6xWDO9vQ=";
```

Reply:
Hey @AndersonTorres!

I tried to understand the whole "when to use hash and when to use sha256" by looking at [the manual for fetchfromgithub](https://nixos.org/manual/nixpkgs/stable/#fetchfromgithub) it just mentions 'sha256' as an argument.

Why is hash preferred?

Reply:
For all intents and purposes, they are interchangeable. 

`hash` parameter allows other hash formats besides SHA256, therefore it is future-proof.
Nonetheless, there is no difference between using `sha256` and `hash` (except some minor annoyances).

On the other hand, some legacy fetchers, like `fetchgit, fetchsvn` &c, weren't updated yet. But they are rarely used, the chance you will come across one of them is small. Just use `hash` and be happy.

About the SRI format, `sha256-<crazy string>=`, it is just a new format for sharing hashes. The new `nixUnstable` tools use it by default.

Reply:
Changed to hash